camp/nfutsum/nfutsum-miniproject/XBox Controller Left and Right bumper/src/main/java/frc/robot/commands/MotorBackwardCommand.java

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) FIRST and other WPILib contributors.
```

Comment:
Can you insert the Bishop Blanchet header?

Reply:
This should apply to any file that you principally authored for BB robotics team.